hugegraph-core/src/main/java/com/baidu/hugegraph/job/algorithm/comm/LouvainTraverser.java

```diff
@@ -321,9 +319,7 @@ private Community wrapCommunity(Vertex v, List<Edge> nbs) {
         return comm;
     }
 
-    private Collection<Pair<Community, MutableInt>> nbCommunities(
-                                                    int pass,
-                                                    List<Edge> edges) {
+    private Collection<Pair<Community, MutableInt>> nbCommunities(int pass, List<Edge> edges) {
```

Comment:
wrap line after `int pass, `?

Reply:
so as this case:  (not modify it):

old:
<img width="795" alt="image" src="https://user-images.githubusercontent.com/17706099/199755660-8574b1f2-c478-4e5a-aafb-499e71dcf22f.png">

new:
<img width="814" alt="image" src="https://user-images.githubusercontent.com/17706099/199755799-e3e2a500-28b5-4da4-8ea6-cab0c3ef85cd.png">